From 68cf564890a36a1a094aa08af5cd34fab24e59d3 Mon Sep 17 00:00:00 2001
From: Christian Brauner <brauner@kernel.org>
Date: Wed, 17 Aug 2022 09:39:25 +0200
Subject: [PATCH 30/45] tree-wide: minimize liburing.h inclusion

because it brings in linux/fs.h and defines struct open_how.

Signed-off-by: Christian Brauner (Microsoft) <christian.brauner@ubuntu.com>
---
 src/lxc/macro.h    | 4 ----
 src/lxc/mainloop.h | 4 ----
 2 files changed, 8 deletions(-)

diff --git a/src/lxc/macro.h b/src/lxc/macro.h
index f00230060..58e3a7f25 100644
--- a/src/lxc/macro.h
+++ b/src/lxc/macro.h
@@ -22,10 +22,6 @@
 
 #include "compiler.h"
 
-#if HAVE_LIBURING
-#include <liburing.h>
-#endif
-
 #ifndef PATH_MAX
 #define PATH_MAX 4096
 #endif
diff --git a/src/lxc/mainloop.h b/src/lxc/mainloop.h
index d6995910a..7d644b756 100644
--- a/src/lxc/mainloop.h
+++ b/src/lxc/mainloop.h
@@ -11,10 +11,6 @@
 #include "hlist.h"
 #include "memory_utils.h"
 
-#if HAVE_LIBURING
-#include <liburing.h>
-#endif
-
 #define LXC_MAINLOOP_ERROR -1
 #define LXC_MAINLOOP_CONTINUE 0
 #define LXC_MAINLOOP_CLOSE 1
-- 
2.37.2

